session.go

```diff
@@ -434,6 +434,11 @@ type Query struct {
 	defaultTimestamp bool
 }
 
+// String implements the stringer interface.
+func (q *Query) String() string {
```

Comment:
Good catch @bfontaine - it was actually a mistake I've made for a few months now.  I mis-interpreted [Effective Go](https://golang.org/doc/effective_go.html#pointers_vs_values) and have been "doing it wrong" until you asked the question :)

I'll fix this.